platform/android/java/build.gradle

```diff
@@ -24,7 +24,7 @@ ext {
     sconsExt = org.gradle.internal.os.OperatingSystem.current().isWindows() ? ".bat" : ""
 
     supportedAbis = ["armv7", "arm64v8", "x86", "x86_64"]
-    supportedTargets = ['release':"release", 'debug':"release_debug"]
+    supportedTargets = ['release':"release", 'debug':"debug"]
```

Comment:
What is this change for?

Reply:
It will be rolled back, it is set so I could build and debug straight from gradle build from android studio

Reply:
(release_debug doesn't build with symbols)

Reply:
Actually, I just took a better look at [`platform/android/detect.py`](https://github.com/godotengine/godot/blob/60ea8aea98e76fd4514c65ca1ebd7f015283e991/platform/android/detect.py#L151) and realized that `release_debug` is very different from `debug` target unlike what I thought.
So I would keep that change as is, since as far as Android studio is concerned, `release` and `debug` should be sufficient. If needed, we can always add an [additional `release_debug` build type for Android studio](https://developer.android.com/studio/build/build-variants#build-types) to match all of Godot's build targets.

Do you mind creating a smaller PR that just update the `debug` target to enable debugging in Android studio.

Reply:
the problem with debbuging is not only this as I stated in the pr, but also the 
``` groovy
buildTypes.all { buildType ->
        buildType.zipAlignEnabled false
        buildType.signingConfig null
    }
``` 
in app build.gradle that I commented out and `doNotStrip '**/*.so'` that I added

so... should I add those to the mini pr, or only this for now? (also should I add the upgraded gradle version which worked for me?)

Reply:
NVM mind for now, opened https://github.com/godotengine/godot/pull/36838